tests/foreman/ui/test_org.py

```diff
@@ -496,26 +505,28 @@ def test_add_subnet(self, subnet_name):
 
         """
         strategy, value = common_locators['entity_deselect']
-        org_name = gen_string('alpha')
-        subnet = entities.Subnet(
-            name=subnet_name,
-            network=gen_ipaddr(ip3=True),
-            mask='255.255.255.0'
-        ).create()
-        self.assertEqual(subnet.name, subnet_name)
         with Session(self.browser) as session:
-            make_org(session, org_name=org_name)
-            self.assertIsNotNone(self.org.search(org_name))
-            self.org.update(org_name, new_subnets=[subnet_name])
-            self.org.search(org_name).click()
-            session.nav.click(tab_locators['context.tab_subnets'])
-            element = session.nav.wait_until_element((strategy,
-                                                      value % subnet_name))
-            self.assertIsNotNone(element)
+            for subnet_name in generate_strings_list():
```

Comment:
Same comment as before about creating only one organization and then adding/removing subnets. Not a blocker for this PR. **NOTE**: this test actually does not create a new organization with the subnet already associated to it, which shows that there's no real value in creating a new org for every iteration imho.